### ISSUE TITLE
[request] mysql-connector-c/8.0

### ISSUE THREAD

**Issue by andre-cunha** [conan-io/conan-center-index#374]
### Package Details
  * Package Name/Version: **mysql-connector-c/8.0**
  * website: https://dev.mysql.com/doc/connector-cpp/8.0/en/

The above mentioned version is newly released by the upstream project and not yet available as a recipe. Please add this version.

**Comment by ericLemanissier** [comment]
this is a new package, MySQL Connector/C++ is not the same thing as MySQL Connector/C or libmysqlclient

**Comment by andre-cunha** [comment]
What i am looking for is MySQL Connector/C++. So is this the right package?

**Comment by ericLemanissier** [comment]
Yes, you should just update the issue title and edit your first message to make it clear.

**Comment by andre-cunha** [comment]
done!

**Comment by hwhsu1231** [comment]
May I ask, when will this package be available from ConanCenter?

**Comment by Croydon** [comment]
Whenever someone does the work and opens a pull request. Contributions are always welcome 🙂
